src/types/service_remote_config.go

```diff
@@ -8,30 +8,22 @@ import (
 // ServiceRemoteConfig represents production specific configuration for an application
 type ServiceRemoteConfig struct {
 	Dependencies map[string]RemoteDependency
-	Environment  map[string]string `yaml:",omitempty"`
-	Secrets      []string          `yaml:",omitempty"`
-	URL          string            `yaml:"url,omitempty"`
-	CPU          string            `yaml:"cpu,omitempty"`
-	Memory       string            `yaml:"memory,omitempty"`
+	Environments map[string]ServiceRemoteEnvironment
+	CPU          string `yaml:"cpu,omitempty"`
+	Memory       string `yaml:"memory,omitempty"`
```

Comment:
Wouldn't CPU and Memory be tweaked per environment ideally as well?

Reply:
I don't think so since these are relative units.

Reply:
What do you mean by relative units? You don't need the same amount of cpu/memory in dev as you might production

Reply:
One reason I thought this made more sense is that the dependency like rds has the same allocated storage on each env. I'd say lets leave as is and we can make this configurable per environment if needed.